util/microservice-sdk/js/README.md

```diff
@@ -6,15 +6,63 @@ It gives utility features that most of the dojot microservices needs to implemen
 __NOTE THAT__ this module does not intend to be a generic microservice SDK, but a
 library scoped to attend the dojot microservices necessities.
 
-## Examples
-
-You can find out how to use each feature in the `examples` directory.
-
-### BackPressure 
+### BackPressure
 
 Before building, check examples/back-pressure/config.js for necessary settings.
 You can build a docker image for the example by running:
 
 ```
-sudo docker build -t sdk-backpressure:example -f examples/back-pressure/Dockerfile  . 
-```
\ No newline at end of file
+sudo docker build -t sdk-backpressure:example -f examples/back-pressure/Dockerfile  .
+```
+
+### Producer
+
+To writer messages into Kafka, you use the Producer class, which is a wrapper over the node-rdkafka Producer.
+
+The following example illustrates how to use the Producer:
+
+```js
+const { Kafka: { Producer } } = require('@dojot/microservice-sdk');
+
+const producer = new Producer({
+  'client.id': 'sample-producer',
+  'metadata.broker.list': 'localhost:9092',
+});
+
+
+producer.connect().then(() => {
+    // The target kafka topic, it be a String
+    const targetTopic = 'producer.example.test';
+
+    //producing message in topic producer.example.test with content Message Example
+    producer.produce(targetTopic, "Message Example");
+  })
+  .catch((error) => {
+    console.error(`Caught an error: ${error.stack || error}`, TAG);
+  });
+
+```
+
+#### Producer Configuration
+
+The following properties can be set for the Producer:
```

Comment:
My suggestion is replacing `The following properties can be set for the Producer` by something like `Any property accepted by the Producer of node-rdkafka can be set. The most relevant are listed below.`